### DISCUSSION TITLE
unwanted flash transition in TileLayer

### DISCUSSION THREAD

**Discussion by acthp** [visgl/deck.gl#7929] [General]
I'm seeing a flash when using TileLayer, where a new tile is initially drawn bright, and smoothly fades to its final color. It looks like a transition, but I haven't enabled any transitions. Also, I can't find any in the code. Is there some default transition? What else could be causing this?

![flash](https://github.com/visgl/deck.gl/assets/494506/543e7e13-3515-48e8-921c-a33470ee8f9c)

**Comment by Pessimistress** [comment]
There are multiple tiles being displayed at that spot. Try [refinementStrategy](https://deck.gl/docs/api-reference/geo-layers/tile-layer#refinementstrategy): no-overlap

**Reply by acthp** [reply]
Thanks, that's better. Is this flash due to a "parent" tile (with transparency) being drawn while waiting for all the "child" tiles? So the parent is only removed after all the child tiles are loaded, creating an additive doubling effect? If so, could the child tiles instead be pre-populated with the data from the parent, and the parent removed immediately, so you get the low-res data until the new tiles load?
